_studio/hevce_hw/h265/src/mfx_h265_encode_hw_utils.cpp

```diff
@@ -1580,9 +1580,10 @@ void MfxVideoParam::SyncMfxToHeadersParam(mfxU32 numSlicesForSTRPSOpt)
     }
 
     m_sps.vui.field_seq_flag = isField();
-    if (IsOn(m_ext.CO.PicTimingSEI))
+    if (m_sps.vui.field_seq_flag
```

Comment:
in such design CO.PicTimingSEI == OFF would be ignored. Need ad input from Olga G. and Sergey P. about this

Reply:
@ogolikov , can you please comment?

Reply:
if CO.PicTimingSEI == OFF, SEI must not be written( may be application writes it itself).
if CO.PicTimingSEI is ON, frame_field_info is written in any case (progressive or field). In progressive case it can be used for doubling, for example.
So previous variant is seems to be correct.

Reply:
The change conforms HEVC specification. In the case of interlace coding PT SEI is a must, otherwise bitstream is incorrect.
CO.PicTimingSEI == OFF should be rejected for interlace.
I will add 'IsOn(m_ext.CO.PicTimingSEI) ||' to the condition.